Added profile not found page

harshmathurx opened the pull request:
As discussed in #430 
I have added a page from the unknown username page and added a new Profile not found page
Please check the screenshots below
![image](https://user-images.githubusercontent.com/58532371/135726060-ab302510-a763-4e6d-a689-d1e87ac624a8.png)

rtynski commented on src/Components/Socials.js:
```diff
@@ -10,19 +10,31 @@ function Socials() {
-  return (
+  if (!profileFound) {
+    return <div style={{ textAlign: 'center' }}>
+      <img src="https://d1aettbyeyfilo.cloudfront.net/eddiejaoude/19083469_1619470266Fpchubber-hi.webp" alt="" style={{ width: '150px' }}/>
```
Move style to class.

rtynski replied:
At least move the url to a variable.

harshmathurx replied:
I'll make those changes

rtynski commented on src/Components/Socials.js:
```diff
@@ -10,19 +10,31 @@ function Socials() {
 
-  return (
+  if (!profileFound) {
+    return <div style={{ textAlign: 'center' }}>
```
Move style to class.

CodingSpecies commented:
> 
> 
> As discussed in #430 I have added a page from the unknown username page and added a new Profile not found page Please check the screenshots below ![image](https://user-images.githubusercontent.com/58532371/135726060-ab302510-a763-4e6d-a689-d1e87ac624a8.png)
> 
> [![](https://camo.githubusercontent.com/76e60919474807718793857d8eb615e7a50b18b04050577e5a35c19421f260a3/68747470733a2f2f676974706f642e696f2f627574746f6e2f6f70656e2d696e2d676974706f642e737667)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/434)

I like the design and set up of this page @harshmathurx

harshmathurx commented:
> > 
> > 
> > As discussed in #430 I have added a page from the unknown username page and added a new Profile not found page Please check the screenshots below ![image](https://user-images.githubusercontent.com/58532371/135726060-ab302510-a763-4e6d-a689-d1e87ac624a8.png)
> > 
> > [![](https://camo.githubusercontent.com/76e60919474807718793857d8eb615e7a50b18b04050577e5a35c19421f260a3/68747470733a2f2f676974706f642e696f2f627574746f6e2f6f70656e2d696e2d676974706f642e737667)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/434)
> 
> I like the design and set up of this page @harshmathurx 

thanks, I appreciate it :)

CodingSpecies commented on src/Components/Socials.js:
```diff
@@ -10,19 +10,31 @@ function Socials() {
-  return (
+  if (!profileFound) {
+    return <div style={{ textAlign: 'center' }}>
+      <img src="https://d1aettbyeyfilo.cloudfront.net/eddiejaoude/19083469_1619470266Fpchubber-hi.webp" alt="" style={{ width: '150px' }}/>
```
Could you perhaps have the photo downloaded, so it is always there?

harshmathurx replied:
I shall

harshmathurx replied:
hey @CodingSpecies, can you suggest in which directory do I place the image?

CodingSpecies replied:
@harshmathurx I would perhaps say make an image folder somewhere, or add the image in the public folder, where the icons are. But this is what I suggest, I am not sure what others would say.

harshmathurx replied:
> @harshmathurx I would perhaps say make an image folder somewhere, or add the image in the public folder, where the icons are. But this is what I suggest, I am not sure what others would say. 

aight, thanks
@rtynski do you have any suggestions about where to place the image?

krishguptadev reviewed (approved):
## LGTM 🚀 

Thank you for your contribution to our project. I think your changes are perfect and shall be merged 🎉 

Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.

Also I had a question? Does this send a HTTP 404 Header?

CodingSpecies reviewed (approved):
@harshmathurx this looks really good, and thank you for taking my idea into consideration!

harshmathurx commented:
> ## LGTM 🚀 
> 
> Thank you for your contribution to our project. I think your changes are perfect and shall be merged 🎉 
> 
> Please continue to watch for issues labelled `help wanted`, as these will be additional opportunities to contribute.
> 
> Also I had a question? Does this send a HTTP 404 Header? 

I will look out for more issues, thanks

no it doesn't send a 404 header

harshmathurx commented:
> @harshmathurx this looks really good, and thank you for taking my idea into consideration! 

you're welcome
happy to contribute :)

harshmathurx commented:
@krishguptadev can you please add `hacktoberfest-accepted` label to this?

Panquesito7 commented on src/Components/Socials.js:
```diff
@@ -10,19 +10,33 @@ function Socials() {
+        <div className="flex-column">
+        <img src='/eddiehub_community_logo.webp' alt="image" style={{ width: '150px' }}/>
+        <h1>Profile not found.</h1>
+        <h1>If you are a new user, please consider registering for LinkFree.</h1>
```
```suggestion
        <h1>If you are a new user, please consider registering at LinkFree.</h1>
```

krishguptadev commented:
> @krishguptadev can you please add `hacktoberfest-accepted` label to this?

I can't do that... But I surely request the maintainers to do that

harshmathurx commented:
@eddiejaoude Can you please review this?
